examples/aides_logement/prologue.catala_fr

```diff
@@ -387,7 +387,7 @@ champ d'application ÉligibilitéAidePersonnaliséeLogement:
 ### Éligibilité aux allocations de logement
 
 ```catala-metadata
-déclaration énumération ÉligibilitéAllocationLogement:
+déclaration énumération TypeÉligibilitéAllocationLogement:
```

Comment:
I suspect that now name collisions between scopes and enum/structs are sanctioned with an error message?

Reply:
Yes! small rework of name maps used in Name Resolution also allowed better handling of variable/subscope and enum/struct name conflicts which are now all properly rejected.